app/helpers/application_helper/button/smart_state_scan.rb

```diff
@@ -10,6 +8,14 @@ def check_smart_roles
     end
   end
 
+  def calculate_properties
+    super
+    if disabled?
```

Comment:
I'd prefer to split this PR into 2 separate ones:
1. Solution of the *Configuration* button's visibility (getting rid of `needs :@record`)
2. Setting `self[:title]` correctly using the changes above.

/cc @yaacov @himdel @romanblanco